### DISCUSSION TITLE
Inconsistent results?

### DISCUSSION THREAD

**Discussion by lovechang1986** [regeirk/pycwt#50] [Q&A]
I copy the code to notebook on my machine and tested.
The result as this png:
![test](https://user-images.githubusercontent.com/8605164/41340240-7b18931a-6f29-11e8-86f0-3727e52e7483.png)
As the official result is:
![sample_nino3](https://user-images.githubusercontent.com/8605164/41340279-94c80e08-6f29-11e8-9747-37a6960a36cd.png)
[pycwt document](https://pycwt.readthedocs.io/en/latest/index.html)

The biggest difference is the minimum value of the original data from document is letter than -2, but my test is greater than -2? Is that why they had inconsistent results?

**Comment by tompc35** [comment]
The difference between the two plots arises in this step of the [tutorial](https://pycwt.readthedocs.io/en/latest/tutorial.html#time-series-spectral-analysis-using-wavelets)

> Optionally, we could also rectify the power spectrum according to the suggestions proposed by Liu et al. (2007)[2]
`power /= scales[:, None]`

Removing this optional step will give the official result for the Nino-3 SST example. However, see [Liu et al. (2007)](https://journals.ametsoc.org/doi/abs/10.1175/2007JTECHO511.1) for a discussion of bias in power spectra given by traditional wavelet analysis. Their Figure 4 specifically addresses the difference between these two plots.
